testcases/InstallUpstreamKernel.py

```diff
@@ -132,9 +132,18 @@ def is_url(path):
                     "grub2-mkconfig  --output=/boot/grub2/grub.cfg")
                 log.debug("Rebooting after kernel install...")
                 self.console_thread.console_terminate()
+                self.prompt = self.cv_SYSTEM.util.build_prompt()
+                self.console_thread.console_terminate()
                 con.close()
-                self.cv_SYSTEM.goto_state(OpSystemState.OFF)
-                self.cv_SYSTEM.goto_state(OpSystemState.OS)
+                for i in range(5):
+                    raw_pty = self.wait_for(self.cv_SYSTEM.console.get_console, timeout=20)
```

Comment:
@PraveenPenguin @abdhaleegit I am adding a timeout to the get_login function. As i am observing the login is stuck in the get_login function for forever time and not coming out, code is not executing after that. So added a timeout and try logging in in a for loop

Reply:
looks good to me